blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: blog
+root: ../../..
+author: Victor (Kwnagchun) Lee
+title: "Korean Translation of Software Carpentry - version 5.2"
+date: 2015-04-16
+time: "13:00:00"
+category: ["Translation", "Version 5.2", "Korea", "Book"]
+---
+<!-- start excerpt -->
+<p> Based on Software Carpentry version 5.2, we translated <a href="http://www.xwmooc.net/swc/">all the lessons</a> in the form of pdf, html, mobi, epub, and azw3. It took almost 6 months: the first half spent on mostly lessons, and the second half spent on remainings. While translating and building various ebook formats, the original book production software of Software Carpentry seemed not to have upfront architectural concerns in terms of CJK problems. Also, we are wondering how to generate various book formats from the current developing lessons. 
+</p>
+<!-- end excerpt -->
+
+<p> As the starting point of Software Carpentry in Korea, we started translating Software Carpentry lessons last October. <a href="http://software-carpentry.org/blog/2014/11/korean-translation.html">Initial draft version</a> has been released last September with the Greg and Raniere helps. The <a href="http://software-carpentry.org/blog/2014/11/korean-translation.html">first Software Carpentry</a> has been approved and held at RAPA with the only translated lessons. </p>
+
+<p> After the fist workshop in Korea, we translated the remaining pieces and attempted to generate various book formats. Unfortunately, there were some issues mostly related to CJK problems. Still there are some issues but learners who can understand Korean can assess the Software Carpentry lessons and related materials through various formats.</p>
```

Comment:
Minor type here: "After the fist workshop" -> "After the first workshop"

I would replace "generate various book formats." with "generate various ebook formats.".

Again, can you replace "CJK problems" with "problems due  Chinese, Japanese, and Korean characters". I needed to search on Wikipedia for CJK.